lib/rules/media-query-no-invalid/__tests__/index.mjs

```diff
@@ -282,3 +282,55 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: [true, { ignoreFunctions: ['theme', 'fooFunction', '/^get.*$/'] }],
+
+	accept: [
+		{
+			code: '@media (min-width: theme(screens.md)) {}',
```

Comment:
Functions could also be used to level: `@media theme(screens.md) {}`
Is that also covered by the current implementation?